ts/src/pro/whitebit.ts

```diff
@@ -538,14 +538,14 @@ export default class whitebit extends whitebitRest {
         }
         const stored = this.orders;
         const status = this.safeInteger (params, 0);
-        const parsed = this.parseWsOrder (data, status);
+        const parsed = this.parseWsOrder (this.extend (data, { 'status': status }));
```

Comment:
@kroitor @frosty00 @carlosmiei 

My gut tells me that you're not going  to like this change

Another option would be to extend the result of `parseWsOrder` with `{status: "status"}`

e.g.
```
const parsed = this.extend (this.parseWsOrder (data), {'status': status})
```
 but then the status won't be available when `parseWsOrder` calls `safeOrder`

The stubs are necessary if were going to separate any logic to the base exchange like in https://github.com/ccxt/ccxt/pull/18082

Reply:
@samgermain actually I think it should be ok, I would rather do this than change the methods' signature